P9Operators/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿
+// Declaring and initializing the secondsTotal variable.
+int secondsTotal = 0;
+
+// Prompting the user for a number of seconds.
+Console.WriteLine("Give me a number of seconds");
+// Taking the inputut and converting it to int32 then assigning it to the variable secondsTotal.
+secondsTotal = Convert.ToInt32(Console.ReadLine());
+
+// Takes seconds total and divides it by mintues in a day. Then assigns the amount of days to the variable days.
+int days = (int)secondsTotal / 86400;
+
+// To get the seconds left we use the modulus operator on the secondsTotal variable and assign it to seconds left.
+int secondsLeft = secondsTotal % 86400;
+
+// Then seconds left are used to calculate how many hours by dividing it by the number of seconds in an hour.
+int hours = (int)secondsLeft / 3600;
+
+// Then we use the modulus operator to get the number of seconds left after we divided my the total amount of seconds in an hour.
+secondsLeft = secondsLeft % 3600;
```

Comment:
Very good that you reuse the variable instead of creating new variables every time :)